src/main/scala/com/github/unchama/seichiassist/listener/PlayerInventoryListener.scala

```diff
@@ -613,5 +614,51 @@ class PlayerInventoryListener(implicit effectEnvironment: EffectEnvironment,
     } else if (topinventory.getTitle == LIGHT_PURPLE.toString + "" + BOLD + "スキルを進化させました") {
       event.setCancelled(true)
     }
+
+  }
+
+  @EventHandler
+  /**
+   * 名義除去システム
+   */
+  def onItemNameRemoveEvent(event: InventoryCloseEvent): Unit = {
+    val player = event.getPlayer match {
+      case p: Player => p
+      case _ => return
+    }
+    val inventory = event.getInventory
+
+    //インベントリサイズが36、あるいはインベントリのタイトルが予期したものでなければ処理を終了させる
+    if (inventory.row != 4 ||  inventory.getTitle != s"$GOLD${BOLD}名義をなくしたいアイテムを投入してください") {
```

Comment:
表記ゆれがめちゃ気になってしまうので...すみません...
```suggestion
    if (inventory.row != 4 ||  inventory.getTitle != s"$GOLD${BOLD}所有者表記を削除したいアイテムを投入してください") {
```

Reply:
本当にありがとうございます..
https://github.com/GiganticMinecraft/SeichiAssist/pull/1164/commits/0e32772faf7b1a1ffde49a2cf18eaf8c90e34167 にて修正しました